docs/optimization.md

```diff
@@ -220,6 +220,28 @@ the benefit of the dynamic batcher and multiple instances is model
 specific, so you should experiment with perf_analyzer to determine the
 settings that best satisfy your throughput and latency requirements.
 
+### Host Policy
```

Comment:
Can you move this to a "## NUMA Optimization" section at the end of the doc.

"Many modern CPUs are composed of multiple cores, memories and interconnects that expose different performance characteristics depending on how threads and data are allocated [cite https://www.kernel.org/doc/html/latest/vm/numa.html]. Triton allows you to set *host policies* that describe this NUMA configuration for your system and then assign model instances to different host policies to exploit these NUMA properties."

Then put the existing text followed by an example that shows setting the numa-node and cpu-cores policies for "gpu_0".